src/components/onboarding/OnboardingPage.tsx

```diff
@@ -0,0 +1,58 @@
+'use client'
+
+import Image from 'next/image'
+import { useRouter } from 'next/navigation'
+
+export default function OnboardingPage() {
+  const router = useRouter()
+
+  const handleGithubConnect = async () => {
+    try {
+      // GitHub OAuth 연동 로직 구현
+      console.log('GitHub 계정 연동 시작')
+
+      // 서버에 GitHub 연동 요청
+      // 연동 성공 시 다음 페이지로 이동
+
+      // 임시로 바로 대시보드로 이동
+      router.push('/dashboard')
+    } catch (error) {
+      console.error('GitHub 연동 오류:', error)
+      // 에러 처리 로직 추가 가능
+    }
+  }
+
+  return (
+    <div className="flex h-full w-full flex-col items-start bg-white px-[60px] pb-[55px]">
+      {/* 그라데이션 배경 */}
+      <div className="absolute bottom-0 left-0 h-[226px] w-full bg-gradient-to-b from-transparent to-slate-100" />
+
+      {/* 메인 컨텐츠 */}
+      <div className="flex h-[689px] w-full max-w-[1280px] flex-col items-center justify-center gap-[35px]">
+        <div className="flex flex-col items-center gap-[17px] py-20">
+          <h1 className="w-full max-w-[928px] text-center text-[52px] leading-[62.05px] font-semibold tracking-widest text-slate-800">
+            GitHub 커밋 기반 맞춤 CS 면접 준비,
```

Comment:
글씨가 밑으로 내려가는데 수정해야할것같음

```suggestion
            GitHub 커밋 기반 맞춤 CS 면접 
                        준비,
```
이렇게 보임임

Reply:
화면 크기가 줄어들때 글씨가 내려간다는 소리..인거지? 그거 일부러 그렇게한건데
반응형 하지 말까